### PR TITLE
seven slot medkits as god intended - removes splints, returns splinting behavior to gauze

### PR DESCRIPTION
**Pull request by Hatterhat**
## About The Pull Request

removes splints. returns splinting behavior to gauze, also preserves splinting overlays:
- gauze acts as splints. you can use gauze where you previously used splints
- if you're gauzing a limb that has both a bleeding and a bonebreak wound, you prioritize the bleed over the bonebreak, and have to use the gauze again to splint
- the splint overlay still exists because i liked it

## How This Contributes To The Skyrat Roleplay Experience
seven slot medkits as god intended

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
  
image
image

</details>

## Changelog

:cl:
del: Splints are gone. This does mean that medkits now start properly full and not overstuffed by one item, at least.
qol: Gauze is now, once again, usable as a splint. If you're gauzing a limb with both a bleed and a fracture, you prioritize the bleed first.
spellcheck: Gauze that reaches a negative absorption value (making them useless/infective) now tells you that it's ruined.
/:cl:

### CODE REVIEW

File: code/game/objects/items/stacks/medical.dm
Comment (softcerv) [inline code/game/objects/items/stacks/medical.dm:1]: You are making a lot of changes here that you could probably make in a modular file. If you are not sure what I mean by this, bring it up in one of the coding channels on the discord and I will try and help you out when I can.

**Comment by softcerv** [comment]
I tagged your PR with the balance tag, because this does affect the balance of medical gameplay by changing the utility provided by gauze.

**Comment by vinylspiders** [comment]
You missed one instance of  `/obj/item/stack/medical/splint/improvised` in  `_maps/RandomRuins/IceRuins/skyrat/icemoon_underground_icewalker_den.dmm`. Causing the CI to fail.

**Comment by YakumoChen** [comment]
take it upstream?

**Comment by Hatterhat** [comment]
> take it upstream?

tg doesnt have splints they just have bandaging cover both

**Comment by lessthnthree** [comment]
I support this, TG has gauze (not splints though) it seems needlessly complex to have our own 'splints' doing the same thing.

**Review by lessthnthree — APPROVED** [review]

**Comment by GoldenAlpharex** [comment]
I'm still not really in support of this, but I guess it's my fault for not requesting changes on it.

**Pull request merged**